### DISCUSSION TITLE
Adding documentation for this to tldr-pages; anything you'd like to change?

### DISCUSSION THREAD

**Discussion by artemislena** [FiloSottile/age#214] [General]
I didn't really find any other way to contact you, so I decided to open an issue here instead… I opened a PR at https://github.com/tldr-pages/tldr/pull/5758 to add documentation for age to tldr. If there's anything you'd like to change or add about it, you might want to have a look over there.

**Comment by FiloSottile** [comment]
Thank you for doing this! I added some comments in a review.
